extending-valtimo/form-flow/form-flow.md

```diff
@@ -1,4 +1,4 @@
-# Form flow
+# Form flow (deprecated)
```

Comment:
deprecated? Did I miss the memo?

Reply:
no, definitely not

From what I can see, this mistake was made in `development/11.2.0` and its now also wrong in `development/11.3.0`. It has to be fixed there so we can merge it back later

Reply:
Ok. Merge this one and I'll see if I can fix that retroactively.